custom_components/tuya_local/devices/holman_wx8_irrigation_controller.yaml

```diff
@@ -0,0 +1,208 @@
+name: Holman WX8
+products:
+  - id: 3c8bxnseywh7radc
+    model: WX8
+    manufacturer: Holman
+primary_entity:
+  entity: sensor
+  name: Watering
+  dps:
+    - id: 123
+      type: boolean
+secondary_entities:
+  - entity: switch
+    name: Zone 1
+    dps:
+      - id: 102
+        type: boolean
+        name: switch
+  - entity: switch
+    name: Zone 2
+    dps:
+      - id: 103
+        type: boolean
+        name: switch
+  - entity: switch
+    name: Zone 3
+    dps:
+      - id: 104
+        type: boolean
+        name: switch  
+  - entity: switch
+    name: Zone 4
+    dps:
+      - id: 104
+        type: boolean
+        name: switch
+  - entity: switch
+    name: Zone 5
+    dps:
+      - id: 106
+        type: boolean
+        name: switch         
+  - entity: switch
+    name: Zone 6
+    dps:
+      - id: 107
+        type: boolean
+        name: switch       
+  - entity: switch
+    name: Zone 7
+    dps:
+      - id: 108
+        type: boolean
+        name: switch
+  - entity: switch
+    name: Zone 8
+    dps:
+      - id: 112
+        type: boolean
+        name: switch
+  - entity: switch
+    name: Start A
+    dps:
+      - id: 113
+        type: boolean
+        name: switch
+  - entity: switch
+    name: Start B
+    dps:
+      - id: 114
+        type: boolean
+        name: switch
+  - entity: switch
+    name: Start C
+    dps:
+      - id: 115
+        type: boolean
+        name: switch 
+  - entity: sensor
+    name: Firmware Version
```

Comment:
I don't think this needs to be a sensor - it will never change and is informational only.  Instead, add it to the primary_entity as
```
    - id: 116
      name: firmware_version
      type: integer
```